geonames/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+
+class Country(models.Model):
+    isocode = models.IntegerField(primary_key=True)
+    iso = models.CharField(max_length=2)
+    iso3 = models.CharField(max_length=3)
+    fips = models.TextField(null=True)
+    name = models.TextField(null=True)
+    capital = models.TextField(null=True)
+    geonameid = models.BigIntegerField(null=True)
+
+
+class Region(models.Model):
+    isocode = models.ForeignKey(Country, db_column='isocode', on_delete=models.PROTECT)
+    regcode = models.TextField()
+    name = models.TextField(null=True)
+    geonameid = models.BigIntegerField(primary_key=True)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['isocode', 'regcode'], name='unique_country_and_region')
+        ]
+
+
+class District(models.Model):
+    name = models.TextField(null=True)
+    geonameid = models.BigIntegerField(primary_key=True)
+
+    isocode = models.IntegerField()
```

Comment:
Shouldn't this be a foreign key to country like the other fields? or it's different from District? Also I see that District is not being used anywhere right now so I am guessing we are saving the information just in case we have a future use?

Reply:
Good point about the foreign key! I think it did just get left out because we aren't using District right now. 

I do think we should do something with that data eventually, though. It adds another layer of what is possible for data analysis and visualization.